src/components/BooksList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = props => {
+  const { books } = props;
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Book ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>
+        {
+          books.map(
+            book => <Book key={book.id} book={book} />,
+          )
+        }
+      </tbody>
+    </table>
+  );
```

Comment:
Hello @Joseph-Burke, Job well-done 👍 

I quote:::
`If defining this function in the BooksList is a hard requirement of the assignment, I will adjust the code, but I would prefer not to since the method I have used achieves the same effect.`

I really do respect and appreciate your PR description.

But we both know that all Microverse students are bind to follow the project requirements and instructions.
Therefore it's a hard requirement, let's implements `handleBookRemove` here 

Thanks for your understanding.

Reply:
@bellom Haha, that's completely fine. I'll get to work on that now.